integration-tests/.pytest.ini

```diff
@@ -20,3 +20,5 @@ markers =
     clp_s: mark tests that use the CLP-S storage engine
     core: mark tests that test the CLP core binaries
     package: mark tests that run when the CLP package is active
+
+BASE_PORT = 55000
```

Comment:
this is a bit unconventional in my opinion. would it be better to
1. Use the env section
    ```
    env =
        CLP_BASE_PORT=55000
    ```
   Then users can override with `BASE_PORT=1234 pytest` in the CLI
2. OR use pytest_addoption for CLI flags
    ```py
    def pytest_addoption(parser):
        parser.addoption("--clp-base-port", action="store", default="55000")
    ```
    Then users can override with `pytest --clp-base-port=1234` in the CLI